src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -18,6 +18,8 @@
     <AndroidGeneratedClassDirectory Condition=" '$(AndroidGeneratedClassDirectory)' == '' ">..\..\src\Mono.Android\obj\$(Configuration)\monoandroid10\android-$(AndroidLatestStablePlatformId)\mcw</AndroidGeneratedClassDirectory>
     <NoWarn>8632</NoWarn>
     <GenerateResxSource>true</GenerateResxSource>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\product.snk</AssemblyOriginatorKeyFile>
```

Comment:
Should this happen in a different PR?

I have a branch where I also added to `Xamarin.Android.Build.Tasks.targets`:
```xml
<_ILRepackArgs>$(_ILRepackArgs) /keyfile:&quot;$(XamarinAndroidSourcePath)product.snk&quot;</_ILRepackArgs>
```

And also changed the `$(AssemblyVersion)`:
```xml
  <Target Name="_SetAssemblyVersion"
      DependsOnTargets="GetXAVersionInfo"
      BeforeTargets="GetAssemblyAttributes">
    <PropertyGroup>
      <AssemblyVersion>$(ProductVersion).$(PackVersionCommitCount)</AssemblyVersion>
      <FileVersion>$(AssemblyVersion)</FileVersion>
      <InformationalVersion>$(AssemblyVersion)</InformationalVersion>
    </PropertyGroup>
  </Target>
```